frontend/src/routes/_layout/profile/index.tsx

```diff
@@ -20,13 +31,24 @@ function RouteComponent() {
 
   const isLoaded = !isLoading && userProfile;
   const hasError = !!error;
+  useEffect(() => {
+    const timer1 = setTimeout(() => {
+      if (!isLoggedIn) {
+        showLoginModal();
+      }
+    }, 100);
+
+    return () => {
+      clearTimeout(timer1);
+    };
+  }, [isInitialized, isLoggedIn, showLoginModal]);
 
   if (hasError) {
     console.error("Error fetching user profile:", error);
   }
 
   return (
-    <div className="min-h-screen bg-white">
+    <div className="bg-white">
```

Comment:
If we can remove the bg-white, we should. Something like bg-white should be applied to routes/_layout.tsx instead